docs/archlinux.md

```diff
@@ -1,16 +1,16 @@
 (This document partly also applies to other Linuxes)
 
-Choosing an installer ISO
+Choosing an installer ISO img
 -------------------------
 
-The last ISO installer version of Archlinux that supports 32-bit is [2017.02.01](https://www.archlinux.org/releng/releases/2017.02.01/). Later versions of the archisos don't work on the v86 emulator because the installer only supports x86_64, not x86 anymore.  For existing Archlinux installations, updates and patches will be done until somewhere around 2018.
+The last ISO image installer version of Arch Linux that supports the 32-bit architecture is version [2017.02.01](https://www.archlinux.org/releng/releases/2017.02.01/). Later versions of the Arch Linux ISO images don't work on the v86 emulator, because the installer only supports the 64-bit architecture. For existing Arch Linux installations, updates and patches will be done until somewhere around 2018.
 
-In the future the community might come up with an alternative distribution based on Archlinux to maintain support for x86. At this point in time [archlinux32](https://mirror.archlinux32.org) seems to work.
+In the future, the community might come up with an alternative distribution based on Arch Linux to maintain support for the 32-bit architecture. At this point in time [archlinux32](https://mirror.archlinux32.org) seems to work.
```

Comment:
It'd probably better to delete the first paragraph, and refer to archlinux32 directly.